src/components/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -356,30 +415,17 @@ export default class PeopleAndRoles extends Vue {
   }
 
   /**
-   * Resets state properties after a change is completed (or to cancel).
-   */
-  private resetData (): void {
-    this.currentOrgPerson = null
-    this.activeIndex = NaN
-    this.showOrgPersonForm = false
-  }
-
-  /**
-   * Determines whether we have the specified role by count and mode.
+   * Gets the Completing Party in a specified org/person list.
+   * @param list The list to search.
+   * @returns The Completing Party if found, otherwise undefined.
    */
-  private hasRole (roleName: Roles, count: number, mode: Modes): boolean {
-    // 1. filter out removed people
-    // 2. filter in people with specified role
-    const orgPersonWithSpecifiedRole =
-      this.getPeopleAndRoles.filter(people => people.action !== ActionTypes.REMOVED)
-        .filter(people => people.roles.some(party => party.roleType === roleName))
-
-    if (mode === Modes.EXACT) {
-      return (orgPersonWithSpecifiedRole.length === count)
-    }
-    if (mode === Modes.AT_LEAST) {
-      return (orgPersonWithSpecifiedRole.length >= count)
-    }
+  private getCompletingParty (list: OrgPersonIF[]): OrgPersonIF {
+    const i = list?.findIndex(orgPerson =>
+      orgPerson.roles.some(role =>
+        role.roleType === Roles.COMPLETING_PARTY
+      )
+    )
+    return (i >= 0) ? list[i] : undefined
```

Comment:
I changed this because `Object.find()` returns the value of the object and we actually want to return the actual object.